mtlpy/templates/inc/footer.html

```diff
@@ -85,8 +84,7 @@
       }(document, 'script', 'facebook-jssdk'));
     </script>
     {% if GOOGLE_ANALYTICS %}
-    <script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
-    </script>
+    <script src="https://www.google-analytics.com/urchin.js" type="text/javascript"></script>
```

Comment:
this remind me that we'll also need to migrate to ssl only when switching to heroku !